services/web/server/src/simcore_service_webserver/projects/projects_nodes_handlers.py

```diff
@@ -191,6 +212,72 @@ async def retrieve_node(request: web.Request) -> web.Response:
     )
 
 
+@routes.post(f"/{VTAG}/projects/{{project_id}}/nodes/{{node_id}}:start")
+@login_required
+@permission_required("project.update")
+async def start_node(request: web.Request) -> web.Response:
+    """Has only effect on nodes associated to dynamic services"""
+    req_ctx = RequestContext.parse_obj(request)
+    path_params = parse_request_path_parameters_as(_NodePathParams, request)
+    try:
+
+        await projects_api.start_project_node(
+            request, req_ctx.user_id, path_params.project_id, path_params.node_id
+        )
+
+        return web.HTTPNoContent(content_type=MIMETYPE_APPLICATION_JSON)
+    except ProjectStartsTooManyDynamicNodes as exc:
+        raise web.HTTPConflict(reason=f"{exc}") from exc
```

Comment:
interesting choice for error code :-)

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409

Reply:
why? not sure I get your point. what would you use instead?